Engine/Component/ComponentCamera.cpp

```diff
@@ -170,6 +172,13 @@ void ComponentCamera::RecordFrame(float width, float height)
 
 	App->renderer->RenderFrame(*this);
 
+	if (App->debug->show_msaa)
+	{
+		glBindFramebuffer(GL_READ_FRAMEBUFFER, fbo);
+		glBindFramebuffer(GL_DRAW_FRAMEBUFFER, msfbo);
+		glBlitFramebuffer(0, 0, width, height, 0, 0, width, height, GL_COLOR_BUFFER_BIT, GL_NEAREST);
```

Comment:
Here the framebuffer that stores the final texture (the one that is being passed to ImGui) is `msfbo`. This is a bit confusing, because multisampled buffer should be the `GL_READ_FRAMEBUFFER` one. Also looking at next commit, I saw that the multisampled texture is stored in `fbo` framebuffer.

So, this should be swapped.